apps/image-editor/src/js/component/resize.js

```diff
@@ -0,0 +1,103 @@
+import Component from '@/interface/component';
+import { componentNames } from '@/consts';
+
+/**
+ * Resize components
+ * @param {Graphics} graphics - Graphics instance
+ * @extends {Component}
+ * @class Resize
+ * @ignore
+ */
+class Resize extends Component {
+  constructor(graphics) {
+    super(componentNames.RESIZE, graphics);
+
+    /**
+     * Current dimensions
+     * @type {Object}
+     * @private
+     */
+    this._dimensions = null;
+
+    /**
+     * Original dimensions
+     * @type {Object}
+     * @private
+     */
+    this._originalDimensions = null;
+  }
+
+  /**
+   * Get current dimensions
+   * @returns {object}
+   */
+  getCurrentDimensions() {
+    const canvasImage = this.getCanvasImage();
+    if (!this._dimensions && canvasImage) {
+      const { width, height } = canvasImage;
+      this._dimensions = { width, height };
```

Comment:
I think it's okay to use it in a single line.
`this._dimensions = canvasImage;`

Reply:
Yes, with this change will get same results. But _dimensions property should has just two property: width and height, and I think canvasImage is big object for this using.